rgbd/src/client_rgbd.cpp

```diff
@@ -15,18 +15,18 @@ ClientRGBD::ClientRGBD() : image_ptr_(nullptr)
 
 ClientRGBD::~ClientRGBD()
 {
-    nh_.shutdown();
 }
 
 // ----------------------------------------------------------------------------------------
 
 bool ClientRGBD::initialize(const std::string& server_name)
 {
+    ros::NodeHandle nh;
```

Comment:
Why did you make this change? What is the advantage of this approach?
What happens if this node handle goes out of scope?

Reply:
Discussed in person. We don't need the node handle for anything else. So then it is better to not use it as a class variable.